kubernetes/gke-base/docs/index.rst

```diff
@@ -224,6 +224,30 @@ Now open a web-browser to http://localhost:12345, and you should see |c3| with y
 Highlights 
 **********
 
+.. _examples-operator-gke-base-configuration:
+
+Service Configurations
+``````````````````````
+
+The |cp| Helm Charts deliver a reasonable base configuration for most deployments.  What is left to the user is the 'last mile' of configuration specific to your environment.  For this demo we specify the non-default configuration in the ``cfg/values.yaml`` file.   The YAML file facilitates a declarative infastructure approach, but can also be useful for viewing non-default configuration in a single place, bootstrapping a new environment, or sharing in general.
+
+The following is an example section of the demo's ``values.yaml`` file showing the |zk| configuration along with a YAML anchor (``<<: *cpImage``) to promote reuse within the YAML file itself.  See the :devx-examples:`values.yaml|kubernetes/gke-base/cfg/values.yaml` for further details.
+
+::
```

Comment:
IMO this is less interesting than the `configOverrides` for setting auto topic creation.  For your consideration: switch out example

Reply:
Nice suggestion.  Staged docs are updated with suggested change now.